coordinator/cluster/config.go

```diff
@@ -33,5 +35,46 @@ func (c *LoadAgentClusterConfig) IsValid(ltConfig loadtest.Config) error {
 	if ltConfig.UsersConfiguration.MaxActiveUsers*len(c.Agents) < c.MaxActiveUsers {
 		return errors.New("coordinator: total MaxActiveUsers in loadTest should not be less than clusterConfig.MaxActiveUsers")
 	}
+
+	ok, err := checkMaxUsersPerTeam(ltConfig, *c)
```

Comment:
This is the natural place where I thought it would fit best. Because this is also where `ltConfig.UsersConfiguration.MaxActiveUsers*len(c.Agents)`  gets checked. Obviously, this check doesn't happen in a simple loadtest via the ltagent, but it seemed cumbersome to pass the `ClusterConfig` there. 

The problem here is that one needs to manually create the user and make the API calls. But even in the loadtest package, there is no admin user, and this would have been needed to be done separately.

LMK what you think.